src/main/java/thePackmaster/powers/summonspack/JinxPower.java

```diff
@@ -0,0 +1,37 @@
+package thePackmaster.powers.summonspack;
+
+import com.evacipated.cardcrawl.mod.stslib.powers.interfaces.OnReceivePowerPower;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import com.megacrit.cardcrawl.powers.ArtifactPower;
+import com.megacrit.cardcrawl.powers.GainStrengthPower;
+import thePackmaster.actions.summonspack.JinxLoseHPAction;
+import thePackmaster.powers.AbstractPackmasterPower;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.att;
+
+public class JinxPower extends AbstractPackmasterPower implements OnReceivePowerPower {
+    public static String POWER_ID = makeID(JinxPower.class.getSimpleName());
+    public static final String NAME = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).NAME;
+    public static final String[] DESCRIPTIONS = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).DESCRIPTIONS;
+
+    public JinxPower(AbstractCreature owner, int amount) {
+        super(POWER_ID, NAME, AbstractPower.PowerType.DEBUFF, false, owner, amount);
+    }
+
+    @Override
+    public boolean onReceivePower(AbstractPower pow, AbstractCreature target, AbstractCreature source) {
+        if (pow.type != AbstractPower.PowerType.DEBUFF || pow.owner == source || pow.ID.equals(GainStrengthPower.POWER_ID)
```

Comment:
```suggestion
        if (pow.type == AbstractPower.PowerType.BUFF || pow.owner == source || pow.ID.equals(GainStrengthPower.POWER_ID)
```
Please don't check like that since you could catch neutral power type which would be bad